packages/manager/modules/sms/package.json

```diff
@@ -47,9 +49,9 @@
     "ovh-angular-pagination-front": "^6.0.0",
     "ovh-angular-responsive-tabs": "^4.0.0",
     "ovh-angular-ui-confirm-modal": "^1.0.2",
+    "ovh-api-services": "^3.31.0",
     "ovh-manager-webfont": "^1.0.2",
     "ovh-ui-kit": "^2.23.1",
-    "ovh-ui-kit-bs": "^2.1.1",
-    "ovh-api-services": "^3.31.0"
+    "ovh-ui-kit-bs": "^2.1.1"
   }
-}
```

Comment:
The `No newline at the end of file` seems to always appear as soon as a new release is triggered.